kratos/python_scripts/python_solver.py

```diff
@@ -140,10 +140,15 @@ def _ImportModelPart(self, model_part, model_part_import_settings):
         if (input_type == "mdpa"):
             problem_path = os.getcwd()
             input_filename = model_part_import_settings["input_filename"].GetString()
+
+            # Setting some mdpa-import-related flags
             import_flags = KratosMultiphysics.ModelPartIO.READ
             if model_part_import_settings.Has("ignore_variables_not_in_solution_step_data"):
                 if model_part_import_settings["ignore_variables_not_in_solution_step_data"].GetBool():
-                    import_flags = KratosMultiphysics.ModelPartIO.IGNORE_VARIABLES_ERROR|KratosMultiphysics.ModelPartIO.READ
+                    import_flags = KratosMultiphysics.ModelPartIO.IGNORE_VARIABLES_ERROR|import_flags
```

Comment:
@jcotela I think this works too

Reply:
yes, I believe this is the same as `import_flags |= KratosMultiphysics.ModelPartIO.IGNORE_VARIABLES_ERROR`.